addons/payment_test/views/payment_test_templates.xml

```diff
@@ -13,6 +13,33 @@
                            placeholder="Test Card Information"/>
                 </div>
             </div>
+            <div class="row mt-8">
+                <div class="form-group col">
+                    <label>
+                        <input type="radio" name="status"
+                            value="done" title="Payment successful" checked="True"/>
+                        Payment successful
+                    </label>
+                </div>
+                <div class="form-group col">
+                    <label>
+                        <input type="radio" name="status" value="pending" title="Payment pending"/>
+                        Payment pending
+                    </label>
+                </div>
+                <div class="form-group col">
+                    <label>
+                        <input type="radio" name="status" value="cancel" title="Payment canceled"/>
+                        Payment canceled
+                    </label>
+                </div>
+                <div class="form-group col">
+                    <label>
+                        <input type="radio" name="status" value="error" title="Payment error"/>
+                        Payment error
+                    </label>
```

Comment:
I was expecting a dropdown instead of multiple checkboxes :thinking: (but you can keep it like that until PO verification).
(And the display is not fabulous, you could probably find a way to improve it :c)
![image](https://user-images.githubusercontent.com/19364378/157663096-98b26101-35ab-451b-b6b2-b83ddff47f50.png)